remoteconfig/types.go

```diff
@@ -0,0 +1,143 @@
+package remoteconfig
+
+import (
+	"time"
+
+	"google.golang.org/api/iterator"
+)
+
+// TagColor represents a tag color
+type TagColor int
+
+// Tag colors
+const (
+	colorUnspecified TagColor = iota
+	Blue
+	Brown
+	Cyan
+	DeepOrange
+	Green
+	Indigo
+	Lime
+	Orange
+	Pink
+	Purple
+	Teal
+)
+
+// Version a Remote Config template version.
+// Output only, except for the version description.
+// Contains metadata about a particular version of the Remote Config template.
+// All fields are set at the time the specified Remote Config template is published.
+// A version's description field may be specified in PublishTemplate calls
+type Version struct {
+	Description    string    `json:"description"`
+	IsLegacy       bool      `json:"isLegacy"`
+	RollbackSource string    `json:"rollbackSource"`
+	UpdateOrigin   string    `json:"updateOrigin"`
+	UpdateTime     time.Time `json:"updateTime"`
+	UpdateType     string    `json:"updateType"`
+	UpdateUser     *User     `json:"updateUser"`
+	VersionNumber  string    `json:"versionNumber"`
+}
+
+// VersionIterator represents the iterator for looping over versions
+type VersionIterator struct{}
+
+// PageInfo represents the information about a Page
+func (it *VersionIterator) PageInfo() *iterator.PageInfo {
+	// TODO
+	return nil
+}
+
+// Next will return the next version item in the loop
+func (it *VersionIterator) Next() (*Version, error) {
+	return nil, nil
+}
```

Comment:
These need to be implemented. Feel free to remove the placeholders if you do not plan to add them as part of this PR

Reply:
Same here. I think this is a good starting point. You can ignore my initial comment.